python/GafferDispatch/LocalDispatcher.py

```diff
@@ -259,8 +259,9 @@ def __doBackgroundDispatch( self, batch ) :
 			taskContext = batch.context()
 			frames = str( IECore.frameListFromList( [ int(x) for x in batch.frames() ] ) )
 
-			args = [
-				"gaffer", "execute",
+			args = [ "gaffer.cmd" ] if os.name == "nt" else [ "gaffer" ]
```

Comment:
I'll give a try to conditionally installing a Linux or Windows version of `gaffer` and see if that works.

Reply:
Unfortunately it needs to be either `.cmd` or `.bat` to launch the script. Otherwise Windows just says "I don't know what to do with this, error." `.cmd ` seems to be ever so slightly more modern so I'm going with that for now.

There may be some incremental improvement to be had when / if we move to Gaffer having its own executable wrapper instead of launching Python directly. But that would also be predicated on the thing that is actually invoked to launch Gaffer is that executable, not the bash / batch script. Which may be a non-starter.